ASM/c/item_table.c

```diff
@@ -328,6 +328,25 @@ item_row_t item_table[] = {
     [0x0117] = ITEM_ROW(0x53,      SILVER_CHEST, 0x41, -1, 0x00A1, 0x01A1, 0x88, no_upgrade, give_small_key, CASTLE_ID, -1, resolve_text_small_keys), // Ganon's Castle Small Key
     [0x0118] = ITEM_ROW(0x53,      SILVER_CHEST, 0x41, -1, 0x913E, 0x01A2, 0x89, no_upgrade, give_small_key, TCG_ID,    -1, resolve_text_small_keys_cmg), // Small Key (Chest Game)
 
+    // 0x0119 to 0x0121 reserved for dungeon rewards (https://github.com/OoTRandomizer/OoT-Randomizer/pull/2034)
+
+    [0x0122] = ITEM_ROW(0x53,      SILVER_CHEST, 0x41, -1, 0x9211, 0x0195, 0x77, no_upgrade, give_small_key_ring, FOREST_ID, true, NULL), // Forest Temple Key Ring (with boss key)
+    [0x0123] = ITEM_ROW(0x53,      SILVER_CHEST, 0x41, -1, 0x9212, 0x0195, 0x77, no_upgrade, give_small_key_ring, FIRE_ID,   true, NULL), // Fire Temple Key Ring (with boss key)
+    [0x0124] = ITEM_ROW(0x53,      SILVER_CHEST, 0x41, -1, 0x9213, 0x0195, 0x77, no_upgrade, give_small_key_ring, WATER_ID,  true, NULL), // Water Temple Key Ring (with boss key)
```

Comment:
Note to self: move these definitions to get item IDs that don't conflict with other open PRs.